### DISCUSSION TITLE
What is the correct way to read values of properties of individual types of objects?

### DISCUSSION THREAD

**Discussion by sedymrak** [microsoft/vscode-cmake-tools#3516] [Q&A] [answered]
CMake has a concept of [properties](https://cmake.org/cmake/help/latest/manual/cmake-properties.7.html).
Does your Visual Studio Code extension facilitate in some way viewing values of these properties?
(Other then getting or setting these properties programatically?)

**Comment by gcampbell-msft** [comment, marked as answer]
We do not have any functionality that explicitly facilities this. We do have the CMake Debugger, which may make things like this easier, but we don't have explicit general support for viewing the values of properties.

**Reply by gcampbell-msft** [reply]
However, if this is a request that you'd like to see in the extension, please make a feature request GitHub issue and we'd be happy to consider it. Thanks!
